matplotcheck/tests/test_base.py

```diff
@@ -51,41 +51,9 @@ def test_options(pt_line_plt):
     plt.close()
 
 
-def test_correct_title(pt_line_plt):
-    """Check that the correct plot title is grabbed from the axis object.
-    Note that get_titles maintains case."""
-
-    assert "Plot Title" in pt_line_plt.get_titles()[1]
-    plt.close()
-
-
-"""DATACHECK TESTS"""
-
-
-def test_assert_xydata_scatter(pt_scatter_plt, pd_df):
-    """Checks points in scatter plot against expected data"""
-    pt_scatter_plt.assert_xydata(pd_df, xcol="A", ycol="B")
-    plt.close()
-
-
-def test_assert_xydata_scatter(pt_scatter_plt, pd_df):
-    """assert_xydata should fail when we change the data"""
-    pd_df["B"][1] += 5
-    with pytest.raises(AssertionError, match="Incorrect data values"):
-        pt_scatter_plt.assert_xydata(pd_df, xcol="A", ycol="B")
-    plt.close()
-
-
-# def test_assert_xydata_timeseries(pt_time_line_plt, pd_df_timeseries):
-#   """Commenting this out for now as this requires a time series data object
-#   this is failing because the time data needs to be in seconds like how
-#   mpl saves it. """
-#     pt_time_line_plt.assert_xydata(pd_df_timeseries,
-#                                    xcol='time', ycol='A',
-#                                    xtime=True)
-
-
-def test_assert_xydata_xlabel(pt_bar_plt, pd_df):
-    pd_df["A"] = pd_df["A"].apply(str)
-    pt_bar_plt.assert_xlabel_ydata(pd_df, xcol="A", ycol="B")
+"""
```

Comment:
ahhhh this test should go in a spatial tests file. @ryla5068 why don't you create the file and add this even if it's commented out ... this way it's not left in base.

Reply:
also @ryla5068 please see above

https://github.com/earthlab/matplotcheck/pull/112/files#diff-8b07e1c1007f53de8610db3fec8039f7R44 
there is a question on line 44 that i didn't notice. can you kindly look into that? if it's not your question, then i think we can just remove it. we did decide to write individual tests which will be easier to manage and fix over time.